Frontend/android-native/app/build.gradle

```diff
@@ -6,6 +6,7 @@ apply plugin: 'com.google.gms.google-services'
 apply plugin: 'com.google.firebase.crashlytics'
 // Apply the Performance Monitoring plugin
 apply plugin: 'com.google.firebase.firebase-perf'
+apply plugin: 'com.google.gms.google-services'
```

Comment:
@ashkritidewan2000 this plugin is already there, why is it added again?

Reply:
Removed